src/lib/components/tablev2/MultiSelectableContent.tsx

```diff
@@ -207,6 +213,9 @@ export const MultiSelectableContent = ({
             hasScrollBar={hasScrollbar}
             scrollBarWidth={scrollBarWidth}
             rowHeight={rowHeight}
+            ref={(node) => {
+              headerRef(node);
+            }}
```

Comment:
nit:
```suggestion
            ref={headerRef}
```

Reply:
Yeah, now that we have it fresh in our minds how passing functions works, this option can be made :D Latest commit fixes this